src/main/java/org/prgrms/kdt/enums/VoucherType.java

```diff
@@ -0,0 +1,26 @@
+package org.prgrms.kdt.enums;
+
+import java.util.Arrays;
+
+public enum VoucherType {
+	FixedAmountVoucher(1),
+	PercentDiscountVoucher(2);
+
+	private final static String UN_SUPPORTED_VOUCHER_MESSAGE = "잘 못된 입력 입니다.";
+	private int voucherIdx;
+
+	VoucherType(int voucherIdx) {
+		this.voucherIdx = voucherIdx;
+	}
+
+	public static VoucherType valueOf(int voucherIdx) {
+		return Arrays.stream(VoucherType.values())
+			.filter(voucher -> voucher.getVoucherIdx() == voucherIdx)
```

Comment:
```suggestion
			.filter(voucher -> voucher.voucherIdx == voucherIdx)
```
p5; 
`VoucherType`안에서는 getter를 사용하지 않아도 내부 필드에 접근이 가능합니다. 
현재는 Console에서 사용하기 위해서 getter를 열었지만 내부에서만 사용할거라면 굳이 `getter`를 열어줄 필요는 없겠죠?